### DISCUSSION TITLE
Max events size cannot be set more than 128

### DISCUSSION THREAD

**Discussion by mounikaVeeha** [open-telemetry/opentelemetry-specification#3315] [Q&A]
Hi , We have a requirement where for each trace id we need to set 1000 events for one trace .But as per the documentation we see that we can only set events size to 128 . Is there a way where we could increase size based on the requirement we have

**Comment by jack-berg** [comment]
128 is the default max number of events. See [these docs](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#span-limits) for info on the defaults and how to change them.
